src/pydisagg/ihme/preprocess.py

```diff
@@ -0,0 +1,417 @@
+from typing import Tuple
+
+import numpy as np
+import pandas as pd
+from typing import Dict, List
+
+from .age_var import (
+    age_id_map,
+    match_pats,
+    match_pops,
+)
+
+
+def validate_data(df, pop_df, age_id_map=age_id_map):
```

Comment:
We should develop some tests for these functions, so that we know we are not messing up anything :)